File: glasses-management-backend-h2/src/main/java/com/glasses/entity/SysUser.java
package com.glasses.entity;

import com.baomidou.mybatisplus.annotation.FieldFill;
import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableLogic;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.util.Date;

@Data
@TableName("sys_user")
public class SysUser {
    @TableId(type = IdType.AUTO)
    private Long id;
    private String username;
    private String phone;
    private String password;
    private String realName;
    private String role;
    private Boolean mustChangePassword;
    private Boolean disabled;
    private Date disabledTime;
    @TableLogic
    private Boolean deleted = false;
    private Date deletedTime;
    @TableField(fill = FieldFill.INSERT)
    private Date createTime;
}

File: glasses-management-android-native/app/src/main/java/com/glasses/app/data/model/SysUser.kt
package com.glasses.app.data.model

data class SysUser(
    val id: Long? = null,
    val username: String? = null,
    val phone: String? = null,
    val realName: String? = null,
    val role: String? = null,
    val mustChangePassword: Boolean? = null,
    val disabled: Boolean? = null,
    val disabledTime: String? = null,
    val deleted: Boolean? = null,
    val deletedTime: String? = null,
    val createTime: String? = null
)
